ocs_ci/ocs/ocp.py

```
@@ -1457,7 +1468,14 @@ def check_phase(self, phase):
                 current_phase = data["status"]["operationState"]["phase"]
             else:
                 current_phase = data["status"]["phase"]
-            log.info(f"Resource {self.resource_name} is in phase: {current_phase}!")
+            last_phase = getattr(self, "_last_logged_phase", None)
+            if current_phase != last_phase:
+                log.info(f"Resource {self.resource_name} is in phase: {current_phase}!")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)